src/screens/Airdrop/RequirementsScreen.tsx

```diff
@@ -0,0 +1,147 @@
+import { StackNavigationProp } from '@react-navigation/stack';
+import React, { Component } from 'react';
+import { Image, Text, StyleSheet, View, TouchableOpacity } from 'react-native';
+import { connect } from 'react-redux';
+
+import { icons } from 'app/assets';
+import { ScreenTemplate, Button, Header } from 'app/components';
+import { Route } from 'app/consts';
+import { ApplicationState } from 'app/state';
+import { actions } from 'app/state/airdrop';
+import { typography, palette } from 'app/styles';
+
+const i18n = require('../../../loc');
+
+interface ActionProps {
+  completeThankYouFlow: Function;
+}
+
+type Props = {
+  navigation: StackNavigationProp<any, Route.AirdropRequirements>;
+} & ActionProps;
+
+class AirdropRequirementsScreen extends Component<Props, {}> {
+  onSoundsGreatPress = () => {
+    this.props.completeThankYouFlow();
+    // this.props.navigation.navigate(Route.Dashboard); WIP
```

Comment:
We don't have any convention agreed upon yet, but personally, I like more TODO comments in cases like this.

Reply:
sure, will change to TODO in that case

Reply:
done